### ISSUE TITLE
Enhancements for Theme Engine (Snygg)

### ISSUE THREAD

**Issue by itsmartashub** [florisboard/florisboard#2330]
I am aware that there are more pressing matters here, but let this suggestion stand for consideration, and perhaps it will be addressed at some point.

Namely, the theme editor you've conceived and created is something truly unique, unlocking creativity, and I thank you all guys for enabling such extensive keyboard personalization.

While creating a Gboard-like UI, I encountered a few things that I couldn't resolve, and it might be useful to add in general:

**1.  Add properties for `padding`, `margin`, `gap`**
- Somewhere, I managed to simulate the gap between elements with `border-width` and `border-color` (like in `action key tails`, `sticky action key`, etc.). However, this doesn't always have the desired impact or doesn't work at all, or the element doesn't support those properties. Besides, using borders for spacing is a bit of a silly solution on its own, I admit 🥲
&nbsp;&nbsp;&nbsp;

**2.  Separate the design of `action keys` from `sticky action key`**
- If a sticky key (e.g., mic🎙️) has a specific design, perhaps someone doesn't want their sticky key to be a mic but incognito or something else, and reordering them using the `Reorder Actions` button, the design remains bland for the mic, and it looks inconsistent when the key with the sticky design is moved to the `action row`. Also, the new key that is now in place of the `sticky action key` does not have the desired sticky design. Huh, if you managed to understand anything of what I said.

![florisboard--sticky-action-key](https://github.com/florisboard/florisboard/assets/44645238/47f89341-2b6f-4184-bb52-16fcf964d74e)
&nbsp;&nbsp;&nbsp;

**3.  Enable change value of `font-size` in `action keys tail`**
- The `font-size` property exists there, but adding any value doesn't seem to have an impact, or perhaps I'm doing something wrong?
&nbsp;&nbsp;&nbsp;

**4.  Perhaps implement `font-weight` property?**
&nbsp;&nbsp;&nbsp;

**5.  Consistent `height`/`padding` in the keyboard footer?**
- I'm not sure what this is related to, but when using a borderless theme, the submit button and space have a smaller height (or padding) than keys `?123`/`ABC` (`-202`/`-201`). While in the theme editor, it looks as it should, but when using the keyboard, somehow the height/padding decreases.

![florisboard--footer-height](https://github.com/florisboard/florisboard/assets/44645238/9bc8b101-a97b-4d7f-8f42-ad0a79f20055)
&nbsp;&nbsp;&nbsp;

**6.  Separate design for all elements of the num layout.**
- Separate design from keys when the num layout is only displayed because if we want to change the design for characters `+`, `(`, `)`, `-`, `%`, `=`, etc., they will be altered on every layout, and that seems odd. But I understand if this might be complicated to implement/rewrite.

![florisboard--num-layout](https://github.com/florisboard/florisboard/assets/44645238/936ce1ba-61d0-4cef-b242-101ebbd1c80a)
![florisboard--num-character](https://github.com/florisboard/florisboard/assets/44645238/b318e3c0-93c5-4579-9308-c09ff9cd1f2f)

&nbsp;&nbsp;&nbsp;

There are some more nuances, but I won't overburden you with them now.

**Comment by atikattar1104** [comment]
I Agree. Some Important CSS Elements Would Definitely Help A Lot In Customizations. 

However, A Manual Editor (Like A Text Editor) For Entire Stylesheet Would Be A Better Option For People With Knowledge Of CSS, As Users Can Directly Use Any CSS Property Of Their Choice, On Any Element Of The Keyboard, And The Built-In CSS Engine Will Do It's Job. This Might Make This Keyboard, The Most Customizable Keyboard On Android. 

BTW, I Love The Theme You Created.
